src/main.cpp

```diff
@@ -192,7 +192,7 @@ int main(int argc, char *argv[]) {
     }
     return 0;
   }
-  a.setQuitOnLastWindowClosed(false);
+  SingleApplication::setQuitOnLastWindowClosed(false);
```

Comment:
QGuiApplication

Reply:
This is buggy, it doesn't work if I change to `QGuiApplication`.